texar/torch/modules/tokenizers/pretrained_bert_tokenizer_utils.py

```diff
@@ -0,0 +1,299 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utils of pre-trained BERT tokenizer.
+"""
+
+from typing import Dict, List, Optional
+
+import collections
+import unicodedata
+
+
+__all__ = [
+    "load_vocab",
+    "BasicTokenizer",
+    "WordpieceTokenizer",
```

Comment:
Are `BasicTokenizer` and `WordpieceTokenizer` specific to BERT or actually reusable in other cases? If the latter, they should not be in "pretrained_bert_tokenizer_utils.py"

Reply:
The functions and classes in `pretrained_bert_tokenizer_utils.py` are only for `BERTTokenizer`, and they will not exposed to the users.